db/schema.rb

```diff
@@ -83,7 +85,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.boolean "enabled", default: false
-    t.decimal "discount_price"
+    t.decimal "discount_price", precision: 8, scale: 2
```

Comment:
schema.rb should never have changes that dont belong in that PR

Reply:
Okay, will take care of this